Cargo.toml

```diff
@@ -6,14 +6,14 @@ members = [
     "c2rust-ast-exporter",
     "c2rust-ast-printer",
     "c2rust-bitfields",
-    "c2rust-macros",
     "c2rust-asm-casts",
     "analysis/runtime",
     "dynamic_instrumentation",
     "pdg",
     "rustc-private-link",
 ]
 exclude = [
+    "c2rust-macros",
     "cross-checks/pointer-tracer",
     "cross-checks/zero-malloc",
     "cross-checks/rust-checks",
```

Comment:
Can do in another PR, but I think we can remove these (as they don't exist anymore) and add `c2rust-refactor` to `exclude`.

Reply:
* `c2rust-refactor` excluded in b0ca01f87ac831d5fa67a561215cc10918e58122
* `cross-checks/*` removed in 3ed73b1bce62fd952e1e5d259598bc7aea50311b